src/Microsoft.AspNetCore.Server.HttpSys/HttpSysOptions.cs

```diff
@@ -81,6 +81,12 @@ public long RequestQueueLimit
             }
         }
 
+        /// <summary>
+        /// The amount of time to wait for active requests to drain while the server is shutting down.
+        /// New requests will receive a 503 response in this time period. The default is 5 seconds.
```

Comment:
remarks looks like it's for extended discussion. summary is for IntelliSense. The comment is short enough we don't need to split it.